.github/workflows/release.yml

```diff
@@ -36,6 +36,18 @@ on:
         description: 'If true, artifacts will be deployed to Nexus.'
         required: false
         default: 'true'
+      rebuildFromExistingTags:
+        description: 'If true, and *forceRecreateTags* is false, a checkout to existing tags will be performed.'
```

Comment:
since we're referring to field names which are not shown in the GH action UI, should we label these with:

`description: 'deployOnNexus: If true, artifacts will be deployed to Nexus.'` 

Same for 

`description: 'forceRecreateTags: If true, tags will be overriden and regenerated. Use with caution.'`